patches/components-os_crypt-keychain_password_mac.mm.patch

```diff
@@ -11,13 +11,13 @@ index 6654c46eb0af784a3a2ff64569d5d1931b9fae30..9640ca797a416a4bd73c1e774b6fb6c8
  #include "base/mac/scoped_cftyperef.h"
  #include "base/rand_util.h"
 @@ -51,8 +52,8 @@ std::string AddRandomPasswordToKeychain(const AppleKeychain& keychain,
- const char KeychainPassword::service_name[] = "Chrome Safe Storage";
- const char KeychainPassword::account_name[] = "Chrome";
+ KeychainPassword::KeychainNameType KeychainPassword::service_name("Chrome Safe Storage");
+ KeychainPassword::KeychainNameType KeychainPassword::account_name("Chrome");
  #else
--const char KeychainPassword::service_name[] = "Chromium Safe Storage";
--const char KeychainPassword::account_name[] = "Chromium";
-+const char KeychainPassword::service_name[] = "Brave Safe Storage";
-+const char KeychainPassword::account_name[] = "Brave";
+-KeychainPassword::KeychainNameType KeychainPassword::service_name("Chromium Safe Storage");
```

Comment:
we may want to file a ticket to utilize the upstream change and remove the patch

Reply:
This change got superseeded by a different approach in https://github.com/brave/brave-core/commit/baa769bab7f1599c3f3f44d8b814d2e1fda30c30